config/application.rb

```diff
@@ -64,16 +64,7 @@ def config.database_configuration
     # Allows us to skip minifincation on some files
     config.assets.skip_minification = []
 
-    # explicitly precompile any images in plugins ( /assets/images ) path
-    config.assets.precompile += [lambda do |filename, path|
-      path =~ /assets\/images/ && !%w(.js .css).include?(File.extname(filename))
```

Comment:
Sure, technically we will link js/css files in `/assets/images`, but is that really a concern?  We can add explicit extensions if needed.